multi/imxrt-multi/config.h

```diff
@@ -26,186 +28,668 @@
 #include "imxrt1060.h"
 #endif
 
+#include <board_config.h>
```

Comment:
The idea is to place all the possible configuration defines in evaluation kits' `board_config.h` (make them "templates"). Peripherals which are not used would also be included (but disabled) and have their default pins defined. Possibly with comments to make it clear how it works.

Reply:
Ok for now, I still lean towards simple description here instead of looking at board_config.h which will also be long - but we will see how it looks when board_config.h would be ready :)